share_extension/android/extension_channels/index.js

```diff
@@ -2,19 +2,19 @@
 // See LICENSE.txt for license information.
 
 import {connect} from 'react-redux';
+import {General} from 'mattermost-redux/constants';
 
 import {
     getExtensionSortedDirectChannels,
-    getExtensionSortedPrivateChannels,
-    getExtensionSortedPublicChannels,
+    getExtensionSortedChannels,
 } from 'share_extension/android/selectors';
 
 import ExtensionChannels from './extension_channels';
 
 function mapStateToProps(state) {
     return {
-        publicChannels: getExtensionSortedPublicChannels(state),
-        privateChannels: getExtensionSortedPrivateChannels(state),
+        publicChannels: getExtensionSortedChannels(state, General.OPEN_CHANNEL),
```

Comment:
I'm not 5/5 about it, but I would say this 2 consecutive calls with different params prevents the memorization of the selector execution. For this cases we usually use a selector "factory". @hmhealey is who always poke me about this kind of things.

Reply:
@hmhealey can you confirm this?

Reply:
actually @jespino you are absolutely right and I had it in a different way before @saturninoabril suggestion, so I'm going to revert that last change